interop-scoring/category-data.json

```diff
@@ -0,0 +1,337 @@
+          "interop-2021-position-sticky"
+        ]
+      },
+      {
+        "name": "interop-2021-transforms",
+        "labels": [
+          "interop-2021-transforms"
+        ]
+      },
+      {
+        "name": "interop-2022-cascade",
+        "labels": [
+          "interop-2022-cascade"
+        ]
+      },
+      {
+        "name": "interop-2022-color",
+        "labels": [
+          "interop-2022-color"
+        ]
+      },
+      {
+        "name": "interop-2022-contain",
+        "labels": [
+          "interop-2022-contain"
+        ]
+      },
+      {
+        "name": "interop-2022-dialog",
+        "labels": [
+          "interop-2022-dialog"
+        ]
+      },
+      {
+        "name": "interop-2022-forms",
+        "labels": [
+          "interop-2022-forms"
+        ]
+      },
+      {
+        "name": "interop-2022-scrolling",
+        "labels": [
+          "interop-2022-scrolling"
+        ]
+      },
+      {
+        "name": "interop-2022-subgrid",
+        "labels": [
+          "interop-2022-subgrid"
+        ]
+      },
+      {
+        "name": "interop-2022-text",
+        "labels": [
+          "interop-2022-text"
+        ]
+      },
+      {
+        "name": "interop-2022-viewport",
+        "labels": [
+          "interop-2022-viewport"
+        ]
+      },
+      {
+        "name": "interop-2022-webcompat",
+        "labels": [
+          "interop-2022-webcompat"
+        ]
+      }
+    ]
+  },
+  "2023": {
+    "categories": [
+      {
+        "name": "interop-2021-aspect-ratio",
+        "labels": [
+          "interop-2021-aspect-ratio"
+        ]
+      },
+      {
+        "name": "interop-2021-position-sticky",
+        "labels": [
+          "interop-2021-position-sticky"
+        ]
+      },
+      {
+        "name": "interop-2022-cascade",
+        "labels": [
+          "interop-2022-cascade"
+        ]
+      },
+      {
+        "name": "interop-2022-dialog",
+        "labels": [
+          "interop-2022-dialog"
+        ]
+      },
+      {
+        "name": "interop-2022-text",
+        "labels": [
+          "interop-2022-text"
+        ]
+      },
+      {
+        "name": "interop-2022-viewport",
+        "labels": [
+          "interop-2022-viewport"
+        ]
+      },
+      {
+        "name": "interop-2022-webcompat",
+        "labels": [
+          "interop-2022-webcompat"
+        ]
+      },
+      {
+        "name": "interop-2023-cssborderimage",
+        "labels": [
+          "interop-2023-cssborderimage"
+        ]
+      },
+      {
+        "name": "interop-2023-color",
+        "labels": [
+          "interop-2022-color",
+          "interop-2023-color"
+        ]
+      },
+      {
+        "name": "interop-2023-container",
+        "labels": [
+          "interop-2023-container"
+        ]
+      },
+      {
+        "name": "interop-2023-contain",
+        "labels": [
+          "interop-2022-contain",
+          "interop-2023-contain"
+        ]
+      },
```

Comment:
For category data, the `name` property is often redundant until categories like this one, which are comprised of multiple labels and the `name` prop is the category's overarching name that is written into the csv.